libs/blocks/iframe/iframe.js

```diff
@@ -0,0 +1,13 @@
+export default function init(a) {
+  const url = a?.href ?? a?.querySelector('a')?.href;
```

Comment:
Nit: The amount of `?` operators in there makes it hard to read in my opinion
```suggestion
export default function init(a) {
  if (!a) return;
  const url = a.href ?? a.querySelector('a')?.href;
```

Reply:
I think we can assume the parameter exists, that is the convention for all other blocks.
```
const url = a.href ?? a.querySelector('a')?.href;
```